animareflection/ui
# Stat Component

## Description

##### Task link: https://trello.com/c/cSSxZRN9/122-stat-component

Added `Stat` component

## Test Steps

1. Ensure the component's structure, stories, and testing suite are robust.
2. Confirm that all tests are successful.
3. Check that the Stat component displays correctly in both storybook and sample downstream applications.
